sdk/eventhub/Directory.Build.props

```diff
@@ -25,7 +25,7 @@
     Azure messaging client libraries.
   -->
   <PropertyGroup Condition="'$(SignAssembly)' == 'true'">
```

Comment:
We should consider also moving this props file under Microsoft.Azure.EventHubs as well. That way it doesn't get picked up by the track 2 and management libraries when they are moved.  It wouldn't be too bad to move this under Microsoft.Azure.EventHubs and then add a Directory.Build.props file in the other 2 package folders that reference this by relative path.

Reply:
Good idea... that hadn't occurred to me.   Done!